lib/fog/azurerm/models/storage/blob.rb

```diff
@@ -5,8 +5,124 @@ class AzureRM
       # delete/destroy for Blob.
       class Blob < Fog::Model
         identity  :name
+        attribute :accept_ranges
+        attribute :cache_control
+        attribute :committed_block_count
         attribute :container_name
+        attribute :content_length
+        attribute :content_type
+        attribute :content_md5
+        attribute :content_encoding
+        attribute :content_language
+        attribute :content_disposition
+        attribute :copy_completion_time
+        attribute :copy_status
+        attribute :copy_status_description
+        attribute :copy_id
+        attribute :copy_progress
+        attribute :copy_source
+        attribute :etag
+        attribute :lease_duration
+        attribute :lease_state
+        attribute :lease_status
+        attribute :last_modified
         attribute :metadata
+        attribute :sequence_number
+        attribute :blob_type
+
+        def save(options = {})
```

Comment:
storage_service.blobs.create() method is not working because 'file_path' is always passed as nil.